managed/ui/src/components/universes/UniverseForm/ClusterFields.js

```diff
@@ -1496,23 +1497,40 @@ export default class ClusterFields extends Component {
         });
     }
 
-    let placementStatus = <span />;
+    let placementStatus, placementStatusOnprem = null;
     const cluster =
       clusterType === 'primary'
         ? getPrimaryCluster(_.get(self.props, 'universe.universeConfigTemplate.data.clusters', []))
         : getReadOnlyCluster(
           _.get(self.props, 'universe.universeConfigTemplate.data.clusters', [])
         );
     const placementCloud = getPlacementCloud(cluster);
-    if (self.props.universe.currentPlacementStatus && placementCloud) {
+
+    // For onprem provider type if numNodes < maxNumNodes then show the AZ error.
+    if ((
+      self.props.universe.currentPlacementStatus && placementCloud
+      && isNonEmptyArray(formValues[clusterType].regionList)
+      && isNonEmptyString(formValues[clusterType].provider))
+    ) {
       placementStatus = (
         <AZPlacementInfo
           placementInfo={self.props.universe.currentPlacementStatus}
           placementCloud={placementCloud}
+          providerCode={currentProvider.code}
+        />
+      );
+    } else if (currentProvider?.code === 'onprem' &&
+      !(this.state.numNodes <= this.state.maxNumNodes)
+      && self.props.universe.currentPlacementStatus
+      && isNonEmptyArray(formValues[clusterType].regionList)
+      && isNonEmptyString(formValues[clusterType].provider)) {
+      placementStatusOnprem = (
```

Comment:
move `) {` to the new line pls for better readability:
```suggestion
      && isNonEmptyString(formValues[clusterType].provider)
    ) {
      placementStatusOnprem = (
```

Reply:
Done